### PR TITLE
add document information to information-security.md

### PR DESCRIPTION
**Pull request by g-may**

### CODE REVIEW

File: information-security.md [hunk 1]
```diff
@@ -45,12 +45,16 @@ Learn more about IBM's own GDPR readiness journey and our GDPR capabilities and
...
 
-### Translation data
+### Translation data and documents
 
 When you send text to {{site.data.keyword.languagetranslatorshort}} in the **Translate** method, the service caches the source text and the translation result to improve performance when it receives the same text in subsequent **Translate** requests. Cached translation text is deleted only after it is not used for a period of 15 days.
 
+Documents that are submitted to the service for translation through the **Translate document** method are stored internally to provide the translated file, and to support additional **Translate document** requests by reference to the original uploaded document. The original document and any associated translated documents are deleted only after they are not used in subsequent **Translate document** requests for a period of 15 days. The **Delete document** method can be used to delete documents before the deletion deadline. 
+
+Translation data and documents are encrypted at rest and on wire when stored and in transit.
```
Comment (rhorsfal) [inline information-security.md]: "on wire" - don't we normally use "in flight" - either way , can we just say "at all times?

File: information-security.md [hunk 2]
```diff
@@ -45,12 +45,16 @@ Learn more about IBM's own GDPR readiness journey and our GDPR capabilities and
...
+
 ### Custom model training data
 
-When you train a custom model, the training data that you use to create the model is stored to provide you with the custom model. The training data persists in storage until the model is deleted with the **Delete model** method.
+When you train a custom model, the training data that you use to create the model is stored to provide you with the custom model. When stored, custom models and training data are encrypted at rest. The training data persists in storage until the model is deleted with the **Delete model** method.
```
Comment (rhorsfal) [inline information-security.md]: Why are we saying this again?

**Comment by reidabook** [comment]
@g-may what is blocking this?

**Comment by reidabook** [comment]
@g-may why is this blocked?

**Pull request merged**